growth/views/firefox_android_clients.view.lkml

```diff
@@ -0,0 +1,137 @@
+view: firefox_android_clients {
```

Comment:
I pulled this view manually because there are a _ton_ of fields in it that I don't want to expose to users.

Reply:
Could we not still do this through the normal automation and then use "refinements" to hide specific dimensions via looker-spoke?

https://cloud.google.com/looker/docs/lookml-refinements

Example:
```
view: +faa_flights {
  final: yes
  dimension: carrier {
    hidden: yes
  }
}
```

Reply:
We _could_, but I found it extremely unwieldy (and hate defining in the negative, so to speak). Let's merge this as-is and I'll think about moving it to the looker-hub defn